tests/framework/matcher/existence.go

```diff
@@ -13,10 +13,6 @@ func Exist() types.GomegaMatcher {
 	return existMatcher{}
 }
 
-func BePresent() types.GomegaMatcher {
-	return existMatcher{}
-}
-
```

Comment:
I usually advocate for keeping matchers since they contribute to cleaner and more readable tests.
However, in this case, the removal is justified as we already have a duplicate matcher right above it.
Thank you!
/lgtm